CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+Breaking changes:
+
+- Enable Server-side Apply by default (https://github.com/pulumi/pulumi-kubernetes/pull/2398)
```

Comment:
not for this PR but won't matter in the end

Reply:
Yeah, I'll make sure this is accurate once all these PRs get merged.